laravel/app/Services/AutoDeliveryService.php

```diff
@@ -54,6 +52,11 @@ public static function DeliveryExecute($ship_date, $order_indexes)
                 $cntend = count($inventories);
                 $cnt = 0;
 
+                if($cntend == 0) {
+                    $lost_point = "在庫無し";
+                    throw new Exception($lost_point);
```

Comment:
一度しか使わない場合は、変数に入れなくても良いかなと思います。
```suggestion
                    throw new Exception("在庫無し");
```